### ISSUE TITLE
Command paths not appropriate for windows & require extension to be installed

### ISSUE THREAD

**Issue by danyill** [griffin-stewie/asciidoctor-preview#31]
The default paths require the asciidoctor-diagram package to be installed and don't function on a Windows platform.

I have removed this package dependency which should either be removed or noted in the documentation (`-r asciidoctor-diagram`) and quoted the paths in `command:default` as well as making the executable run `asciidoctor.bat`. I have also changed `path:default` to refer to the Ruby interpreter.

I can successfully run on Windows using the following:

```
module.exports =
  config:
    command:
      title: 'Command'
      description: 'You can use {{{tempHTMLPath}}}, {{{baseDirPath}}}, {{{filePath}}}'
      type: 'string'
      default: "asciidoctor.bat --safe-mode unsafe -a lang=ja -b html5 -d book --base-dir \"{{{baseDirPath}}}\" -o \"{{{tempHTMLPath}}}\" \"{{{filePath}}}\""
      order: 40
    path:
      title: 'PATH'
      description: 'for commands that you use'
      type: 'string'
      default: 'C:\\Ruby23\\bin\\'
      order: 41
```

**Comment by griffin-stewie** [comment]
@danyill 
Thank you for your report.

I didn't know that asciidoctor-diagram doesn't work on Windows.

First of all, this package's main feature is that we can preview with asciidoctor-diagram. I can say that is only point compare with [asciidoc-preview](https://atom.io/packages/asciidoc-preview). So if you don't need asciidoctor-diagram, I recommend to use [asciidoc-preview](https://atom.io/packages/asciidoc-preview) instead my package.

Basically I don't support Windows platform, because I don't have a Windows machine.

I think I should write these things on README.md. and put a setting example you discovered as well.
